frontend/src/components/Following/FollowingButton.js

```diff
@@ -0,0 +1,67 @@
+import '../../config';
+import React, { useState, useEffect } from 'react';
+import axios from "axios";
+
+// Stylesheet
```

Comment:
I understand the use of this comment but since the import is rather self-explanatory, I would remove this comment

Reply:
I just think it looks neater sectioned off like that, especially since we have a lot of imports in a lot of files. For this file, it isn't too bad, but some have several and this would make them consistent across the board. If you feel strongly about it I can remove it, but I feel like just sepe\arating them by a blank line feels messy and so does not separating them at all.